aggregations/db_tables/daily_accounts_added_per_entity.py

```diff
@@ -0,0 +1,98 @@
+from . import DAY_LEN_SECONDS, daily_start_of_range
+from ..periodic_aggregations import PeriodicAggregations
+
+"""
+This code builds a table intended to assist with calculating statistics
+relating accounts ("added_accounts") to ecosystem entities ("entities")
+and is originally intended to be used for tracking new accounts added to each entity.
+"""
+
+
+class DailyAccountsAddedPerEntity(PeriodicAggregations):
+    def dependencies(self) -> list:
+        return ["near_ecosystem_entities"]
+
+    @property
+    def sql_create_table(self):
+        return """
+            CREATE TABLE IF NOT EXISTS daily_accounts_added_per_entity
+            (
+                entity_id                           TEXT NOT NULL,
+                account_id                          TEXT NOT NULL,
+                added_at_block_timestamp            BIGINT NOT NULL,
```

Comment:
Given that nearcore timestamp type is unsigned 64-bit integer, we might not fit into BIGINT (signed 64-bit integer [not any time soon, but I prefer to be consistent across the ecosystem]), so following the types we use in Indexer for Explorer and also in `deployed_contracts` table in near-analytics, let's use `numeric(20, 0)`:

```suggestion
                added_at_block_timestamp            numeric(20, 0) NOT NULL,
```